src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java

```diff
@@ -0,0 +1,221 @@
+package com.denizenscript.depenizen.bukkit.commands.quests;
+
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+
+import me.blackvein.quests.Quest;
+import me.blackvein.quests.Quester;
+import me.blackvein.quests.Quests;
+import me.blackvein.quests.exceptions.InvalidStageException;
+import me.blackvein.quests.util.Lang;
+
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+
+import org.bukkit.entity.Player;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.debugging.Debug;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.depenizen.bukkit.bridges.QuestsBridge;
+
+/**
+ * Command integration for Quests
+ * @author PikaMug
+ * @since 2.0.0-b595
+ */
+public class QuestsCommand extends AbstractCommand {
+
+    // <--[command]
+    // @Name Quests
+    // @Syntax quests [add/remove/set] (questId:<quest-id>) (stage-no:<#>) (points:<#>) (state:true/false)
+    // @Group Depenizen
+    // @Plugin Depenizen, Quests
+    // @Required 1
+    // @Short Edits quest player information.
+    //
+    // @Description
+    // This command allows you to give, quit, or set the stage of a quest for a player, or to add, subtract, or set
+    // the amount of Quest Points that a player holds.
+    //
+    // @Tags
+    // None
+    //
+    // @Usage
+    // Use to force player to take MyFirstQuest quest, ignoring requirements.
+    // - quests add quest-id:custom1 state:true
+    //
+    // @Usage
+    // Use to force player to quit MyFirstQuest quest, notifying player.
+    // - quests remove quest-id:custom2 state:true
+    //
+    // @Usage
+    // Use to force player into specified stage 2 of MyFirstQuest quest.
+    // - quests set quest-id:custom3 stage-no:2
+    //
+    // @Usage
+    // Use to give player 100 Quest Points.

```

Comment:
As implied in my other comment, `the linked player` or `the player` or something, as `Use to give player something` isn't grammatically correct.

Reply:
Addressed